app/src/main/java/com/hedvig/app/feature/home/ui/HomeAdapter.kt

```diff
@@ -107,6 +114,9 @@ class HomeAdapter(
         if (holder is ViewHolder.ClaimStatus) {
             holder.composeView.disposeComposition()
         }
+        if (holder is ViewHolder.InfoCard) {
+            holder.composeView.disposeComposition()
+        }
```

Comment:
In other adapters like the `OfferAdapter` we're using a function that looks like this:
```kotlin
override fun onViewRecycled(holder: ViewHolder) {
    val itemView = holder.itemView
    if (itemView is ComposeView) {
        itemView.disposeComposition()
    }
}
```

Is it something that could be used universally in such situations maybe? To avoid having to manually add all cases when we add more composable ViewHolders

Reply:
Agreed!